### PR TITLE
[WIP] init v3.x.x focus-components

### LINKED ISSUE #1004: [Libs] migrate to the last version of lodash
Would be useful to develop components with the latest version of lodash.

### PR DESCRIPTION
**Pull request by TomGallon**
This branch init focus-components v3.x.x

# TODO v3.x.x

### Global tests
- [ ] add tests on all components

### New components
- [x] ajouter chips
- [x] ajouter select material
- [x] remplacer dropdown
- [ ] add this 3 components to the showcase

### Application
- [x] Popin
- [x] Confirmation-popin
- [x] Menu

## List ?
- [ ] add component table
- [ ] add component list
- [ ] add component timeline
- [ ] wrapper scroll infini

## search
- [ ] quick search ?

## Behaviours to rewrite
- [ ] Grid
- [ ] Material
- [ ] Scroll
- [x] Translation

## Build
- [ ] Epurer la conf webpack
- [ ] Revoir les tâches de build dans package.json pour réécrire la tâche avec babelify
- [x] mettre en place les peer dependencies

## Showcase
- [ ] Rewrite the Showcase and complete it
- [ ] Add all missing components

## Libs to upgrade
- [x] moment
- [x] lodash [fixes #1004]
- [x] i18n

### CODE REVIEW

**Comment by pierr** [comment]
2 things done and already 15k lines of code in less

**Comment by TomGallon** [comment]
fixes https://github.com/KleeGroup/focus-components/issues/1004

File: src/panel/index.js [hunk 1]
```diff
@@ -39,7 +41,7 @@ class Panel extends PureComponent {
             <div className='mdl-card mdl-card--border mdl-shadow--4dp' data-spy={this.spyId} data-focus='panel'>
                 <div className='mdl-card__title mdl-card--border' data-focus='panel-title'>
                     {title && <h3 data-spy-title>{i18next.t(title)}</h3>}
-                    {shouldDisplayActionsTop && <div className='buttons'>{Buttons}</div>}
+                    {shouldDisplayActionsTop && <Buttons editing={editing} toggleEdit={toggleEdit} getUserInput={getUserInput} save={save}/>}
```
Comment (TomGallon) [inline src/panel/index.js]: @Ephrame if you declare Buttons directly a components, you remove the ability to user to define is own actions.

**Comment by pierr** [comment]
:+1:

**Comment by pierr** [comment]
http://www.lovethispic.com/image/35069/home-alone

**Pull request merged**